src/directives/attributes.js

```diff
@@ -0,0 +1,149 @@
+goog.provide('ngeo.AttributesController');
+goog.provide('ngeo.attributesDirective');
+
+goog.require('ngeo');
+goog.require('ngeo.EventHelper');
+
+
+/**
+ * Directive used to render the attributes of a feature into a form.
+ * Example:
+ *
+ * Todo...
+ *
+ * @htmlAttribute {Array.<ngeox.Attribute>} ngeo-attributes-attributes The
+ *     list of attributes to use.
+ * @htmlAttribute {ol.Feature} ngeo-attributes-feature The feature.
+ * @return {angular.Directive} The directive specs.
+ * @ngInject
+ * @ngdoc directive
+ * @ngname ngeoAttributes
+ */
+ngeo.attributesDirective = function() {
+  return {
+    controller: 'ngeoAttributesController',
+    scope: true,
+    bindToController: {
+      'attributes': '=ngeoAttributesAttributes',
+      'feature': '=ngeoAttributesFeature'
+    },
+    controllerAs: 'attrCtrl',
+    templateUrl: ngeo.baseTemplateUrl + '/attributes.html'
+  };
+};
+
+ngeo.module.directive('ngeoAttributes', ngeo.attributesDirective);
+
+
+/**
+ * @param {!angular.Scope} $scope Angular scope.
+ * @param {ngeo.EventHelper} ngeoEventHelper Ngeo event helper service
+ * @constructor
+ * @ngInject
+ * @ngdoc controller
+ * @ngname ngeoAttributesController
+ */
+ngeo.AttributesController = function($scope, ngeoEventHelper) {
+
+  /**
+   * The list of attributes to create the form with.
+   * @type {Array.<ngeox.Attribute>}
+   * @export
+   */
+  this.attributes;
+
+  /**
+   * The feature containing the values.
+   * @type {ol.Feature}
+   * @export
```

Comment:
I think it is.  `this.feature` gets initialized by the `bindToController`, so unless we export `this.feature`, it would be renamed and the rest of the code would fail.

Something we could do is make the inner property private:

```
 /**
  * The feature containing the values.
  * @type {ol.Feature}
  * @private
  */
this.feature_ = this['feature'];
```

but then again the inner property "feature" gets set already... so why not use it ?

Bottom-line: I suggest we leave this as-is.  What do you think ?